src/core/__tests__/props-updating.test.tsx

```diff
@@ -619,3 +623,123 @@ describe("mutation detection", () => {
         expect(Widget.option.mock.calls[0][1]).toEqual(value);
     };
 });
+
+describe("onXXXChange", () => {
+    it("should call on component changes option", () => {
```

Comment:
```suggestion
    it("is called on component changes option", () => {
```
Same for other tests